buildtools/AssemblyCheck/AssemblyCheck.fsproj

```diff
@@ -7,10 +7,16 @@
     <UseAppHost Condition="'$(DotNetBuildFromSource)' == 'true'">false</UseAppHost>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <PublishReadyToRun>true</PublishReadyToRun>
+    <RuntimeIdentifiers Condition="'$(OS)' == 'Unix'">linux-x64</RuntimeIdentifiers>
```

Comment:
This will need Mac as well, right?

Reply:
Hmm, that is a super great question.  Good eyes mate.

It works but I imagine on an M1 it runs emulated rather than arm native.
I think I should expand the palette of options trying to make it native as much as possible.

Reply:
@baronfel just pointed me at the  UseCurrentRuntimeIdentifier property which eliminates all thought and just does 'the right thing (TM)'.

I am a happy bunny